XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/Simulation/Hands/SimulatedHandControllerState.cs

```diff
@@ -0,0 +1,218 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using XRTK.Definitions.Controllers.Simulation;
+using XRTK.Definitions.Utilities;
+using XRTK.Providers.Controllers.Hands;
+using XRTK.Services;
+
+namespace XRTK.Providers.Controllers.Simulation.Hands
+{
+    /// <summary>
+    /// Internal class to define current hand data and smoothly animate hand data points.
+    /// </summary>
+    public class SimulatedHandControllerState
+    {
+        private readonly SimulatedControllerDataProviderProfile profile;
+        private readonly Camera playerCamera;
```

Comment:
This is also troubling as we have a way to get the current camera rig now.

Reply:
`MixedRealityToolkit.CameraSystem.CameraRig.PlayerCamera` you mean this? That's how I am getting it. But I guess you are suggesting not caching the reference here.

Reply:
Ah, so it's just cached. Yeah I'd like to just reference that rig camera directly. We're already caching it there.